src/redux/store/initialized.ts

```diff
@@ -0,0 +1,19 @@
+import { AnyAction } from 'redux';
+
+const SET_INITIALIZED = 'SET_INITIALIZED';
+
+export const setInitialized = (value: boolean) => ({
+  type: SET_INITIALIZED,
+  value,
+});
+
+const initializedReducer = (error = false, action: AnyAction) => {
```

Comment:
you store function in this variable, that means the name should be a verb

Reply:
oooops (copy-past from errorReducer :D)
✅